docs-src/security-model.md

```diff
@@ -22,25 +22,17 @@ security model:
   GatewayClass.
 
 ## Roles
-For the purposes of this security model, 3 common roles have been identified:
-
-* **Infrastructure provider**: The infrastructure provider (infra) is
-  responsible for the overall environment that the cluster(s) are operating in.
-  Examples include: the cloud provider (AWS, Azure, GCP, ...), the PaaS provider
-  in a company.
-* **Cluster operator**: The cluster operator (ops) is responsible for
-  administration of entire clusters. They manage policies, network access,
-  application permissions.
-* **Application developer**: The application developer (dev) is responsible for
-  defining their application configuration (e.g. timeouts, request
-  matching/filter) and Service composition (e.g. path routing to backends). 
-
-Although these roles can cover a wide variety of use cases, some organizations
-may be structured slightly differently. Many organizations may also have a
-fourth role that sits between "cluster operator" and "application developer":
-
-* **Application admin**: The application admin has administrative access to some
-  namespaces within a cluster, but not the cluster as a whole.
+
+There are 3 primary roles in the API:
+
+- Infrastructure provider
+- Cluster Operator
+- Application Developer
+
+There could be a fourth a role of Application Admin in some use cases.
+
+Please refer to [roles and personas](concepts.md#roles-and-personas) section
+in API concepts for details.
```

Comment:
Thanks for cleaning up the duplication between this and the API concepts pages. I'm actually wondering if we should switch these around though. This PR adds all the detail around roles to the API concepts page and leaves a short summary here - maybe the short summary should be in API concepts and the detailed overview should be here. Either way, it's better to not have the full detail in both places, so thanks for fixing that.

Reply:
SGTM. I've updated to switch this around.